src/app/screens/History/index.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import { connect, useSelector } from 'react-redux';
+import UTLabel from '@widergy/energy-ui/dist/components/UTLabel';
```

Comment:
import { UTLabel } from '@widergy/energy-ui' is preferred so you can import multiple components in one line